### ISSUE TITLE
Support translation of exceptions to dep5 exception format

### ISSUE THREAD

**Issue by tenzap** [rpavlik/spdx-to-dep5#3]
Just saw this on lintian and thought this might be also supported by your tool.

https://lintian.debian.org/tags/bad-exception-format-in-dep5-copyright

For now your code doesn't produce such license identifiers because of https://github.com/rpavlik/spdx-to-dep5/issues/2

Two samples:
```
W: qt-android-6.4 source: bad-exception-format-in-dep5-copyright gpl-2.0 with linux-syscall-note [debian/copyright:519]
W: qt-android-6.4 source: bad-exception-format-in-dep5-copyright gpl-3.0-or-later with bison-exception-2.2 
```

**Comment by rpavlik** [comment]
yeah I was just thinking about that last night, we need a translation table. We could test it with a manually fixed SPDX input file: reuse isn't the only ones to generate them.
